<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uploadWizard</class>
 <widget class="QWizard" name="uploadWizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shared Folder Setup Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <widget class="QWizardPage" name="computerPage">
   <property name="title">
    <string>This Computer</string>
   </property>
   <attribute name="pageId">
    <string notr="true">0</string>
   </attribute>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>This Computer Name</string>
      </property>
      <property name="buddy">
       <cstring>nameField</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="nameField"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_8">
      <property name="toolTip">
       <string>Folder to put the shared folder into.</string>
      </property>
      <property name="text">
       <string>Folder to sync:</string>
      </property>
      <property name="buddy">
       <cstring>folderField</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="folderField">
        <property name="toolTip">
         <string>Folder to put the shared folder into.</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhUrlCharactersOnly</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="folderSelectButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>New Share name:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="serverPage">
   <property name="title">
    <string>Server Info</string>
   </property>
   <attribute name="pageId">
    <string notr="true">1</string>
   </attribute>
   <layout class="QFormLayout" name="formLayout_3">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Server Host Name</string>
      </property>
      <property name="buddy">
       <cstring>hostField</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="hostField">
        <property name="inputMethodHints">
         <set>Qt::ImhPreferLowercase</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Port</string>
        </property>
        <property name="buddy">
         <cstring>portField</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="portField">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
        <property name="value">
         <number>22</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Username</string>
      </property>
      <property name="buddy">
       <cstring>usernameField</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="usernameField">
      <property name="inputMethodHints">
       <set>Qt::ImhDialableCharactersOnly</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>nameField</tabstop>
  <tabstop>folderSelectButton</tabstop>
  <tabstop>folderField</tabstop>
  <tabstop>hostField</tabstop>
  <tabstop>portField</tabstop>
  <tabstop>usernameField</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
